src/pages/ScholarshipForm.jsx

```diff
@@ -60,21 +82,7 @@ function ScholarshipForm() {
 
   return (
     <Container maxWidth="md">
-      <Snackbar
-        open={submissionAlert}
-        autoHideDuration={5000}
-        onClose={() => setSubmissionAlert(false)}>
-        <MuiAlert
-          elevation={6}
-          variant="filled"
-          onClose={() => setSubmissionAlert(false)}
-          severity={error ? 'error' : 'success'}>
-          {error
-            ? 'Something went wrong. Please try again later.'
-            : 'Submission successful!'}
-        </MuiAlert>
-      </Snackbar>
-
+      <Fade in={showAlert}>{submissionAlert || <Alert>NA</Alert>}</Fade>
```

Comment:
I'm curious, what does this look like when you just don't render the dummy **Alert**? I'm curious if **Fade** or another transition would gracefully handle the sudden state change from object to null/empty string well.

Reply:
Without the Fade it shifts down everything below it. What I could do is make room for it with a dummy Alert showing nothing and then replace it when there is an actual Alert. I think that should prevent it shifting the elements.

Reply:
Hmm, what you could also do if you want to use other transitions to avoid shifting of the form is show the alert at the very bottom beneath the Submit button. That way it doesn't affect the elements above it